src/_scss/pages/homepageUpdate/heroUpdate/heroUpdate.scss

```diff
@@ -34,7 +34,16 @@
 
 .homepage-hero-graphic-container {
     width: 140%;
-    margin: 0 -250px;
+
+    @media(min-width:768px) and (max-width:992px) {
+        margin-top: 40px;
+    }
+    @media(min-width:992px) {
```

Comment:
you're setting margin-top twice at min 992; could probably condense these last two into one media query